swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/TunnelStore.swift

```diff
@@ -79,16 +78,17 @@ public final class TunnelStore: ObservableObject {
     setupTunnelObservers()
 
     Task {
-      // loadAllFromPreferences() returns list of tunnel configurations we created. Since our bundle ID
-      // can change (by us), find the one that's current and ignore the others.
+      // loadAllFromPreferences() returns list of tunnel configurations created by our bundle ID.
+      // Since our bundle ID can change (by us), find the one that's current and ignore the others.
```

Comment:
It can't change at runtime, right? This is talking about the ID changing after an update is installed?

The App Store doesn't use the bundle as a package ID? Is such a thing possible?

Reply:
Nah, this can't change at runtime. It's inexorably linked to other things in the App Store listing, so it wouldn't be easy to change.

One scenario we may want to change it though is if we build a different Network Extension for some reason (different transport, DNS proxy, content filter, etc) and want to include both or update this one's name.